catboost/private/libs/options/enums.h

```diff
@@ -229,6 +229,7 @@ enum class EFstrType {
     Interaction,
     InternalInteraction,
     ShapValues,
+    ApproximateShapValues,
```

Comment:
Do not change the order.

Reply:
Moved to the end of enums. Also moved calcType to the end of function inputs